ddl/ddl_api.go

```diff
@@ -1228,7 +1253,7 @@ func (d *ddl) AlterColumn(ctx context.Context, ident ast.Ident, spec *ast.AlterT
 		return infoschema.ErrTableNotExists.GenByArgs(ident.Schema, ident.Name)
 	}
 
-	colName := spec.NewColumn.Name.Name
+	colName := spec.NewColumns[0].Name.Name
```

Comment:
Why here doesn't use `changedNewColumn`? And In this function, `modifiedNewColumn` is better.
Besides, `modifiedNewColumn` and `newmodifiedCol`(in line1155) this two styles，you'd better choose a style.